apps/css-workshop/src/components/theme.js

```diff
@@ -143,15 +146,19 @@ class ThemeButton extends HTMLElement {
   }
 
   changeTheme = ({ target: { value: _theme } }) => {
+    const root = document.body;
+    const v5Root = document.documentElement;
```

Comment:
Similar comment here as Ladle: can we set all attributes on the same element (`<body>` or `<html>`) instead of mixing and matching?